analyzer/src/main/kotlin/managers/Conan.kt

```diff
@@ -283,15 +276,19 @@ class Conan(
         }[field].textValueOrEmpty()
 
     /**
-     * Return the full list of packages, excluding the project level information.
+     * Find the node that represents the project defined in the definition file.
      */
-    private fun removeProjectPackage(rootNode: JsonNode, definitionFile: File): List<JsonNode> =
-        rootNode.find {
+    private fun findProjectNode(pkgInfos: JsonNode, definitionFileName: String): JsonNode =
+        pkgInfos.first {
             // Contains because conanfile.py's reference string often includes other data.
-            it["reference"].textValueOrEmpty().contains(definitionFile.name)
-        }?.let { projectPackage ->
-            rootNode.minusElement(projectPackage)
-        } ?: rootNode.toList<JsonNode>()
+            it["reference"].textValueOrEmpty().contains(definitionFileName)
```

Comment:
Nit: Maybe replace `contains` with `in` while at it? (And eventually adjust the comment.)